integration-tests/solclient/solclient.go

```diff
@@ -485,3 +488,64 @@ func (c *Client) GasStats() *blockchain.GasStats {
 func (c *Client) AddHeaderEventSubscription(key string, subscriber blockchain.HeaderEventSubscription) {
 	panic("implement me")
 }
+
+func SendFunds(senderPrivateKey string, receiverPublicKey string, lamports uint64, rpcClient *rpc.Client, wsClient *ws.Client) error {
+
```

Comment:
is there a method to recover funds from the CL nodes once the test wraps up?

Reply:
I tried implementing it but it will require a bit more time, having issues decoding the private key. Create a Jira for it so I tackle it separately